### ISSUE TITLE
Vitamin D should display mcg to conform to 2018 standard

### ISSUE THREAD

**Issue by Cgettys** [nutritionix/nutrition-label#75]
Hello,

I work with @JCapriotti, and was looking at your nutrition label. I appreciate the fast response to #71 , and I hate to bother you guys again. However, I noticed a few things (some of which relate to issue #71) , and I thought you would want to be informed of them.
1. While the old daily value for Vitamin D was 400 IU ([source](https://www.fda.gov/Food/GuidanceRegulation/GuidanceDocumentsRegulatoryInformation/LabelingNutrition/ucm064928.htm)), this will change in 2018. In 2018, the new value will be 20mcg or 800 IU ([source 2](https://www.fda.gov/downloads/Food/GuidanceRegulation/GuidanceDocumentsRegulatoryInformation/LabelingNutrition/UCM513817.pdf)). Of course, if your nutritionist could confirm this, it would be greatly appreciated. 
2. Additionally, according to that second FDA document I linked to above, "The amount of vitamin D may, but is not required to, be expressed in international units (IU), in addition to the mandatory declaration in mcg. Any declaration of the amount of vitamin D in IU must appear in parentheses after the declaration of the amount of vitamin D in mcg.". So, the current label does not conform to the standard.

3. Lastly, if you compare the two sources above, some of the other daily values appear to have been updated. I'm not sure what the best way to handle this is, since this means you might end up with two sets of daily values to keep track of.

To summarize, as I understand it:
The old daily value was 400 IU, and the unit used was IU
The new daily value is 20mcg, which is equal to 800IU. The label is required to use mcg, though it optionally may additionally include the amount in IU in parantheses.
Some other daily values appear to have changed, and you may want to have your nutritionist verify that all your daily values will be up to date.

Some examples of the new label can be found [here](https://www.fda.gov/downloads/Food/GuidanceRegulation/GuidanceDocumentsRegulatoryInformation/LabelingNutrition/UCM511964.pdf).

**Comment by majin22** [comment]
noted and I will forward this to our nutritionist. thanks

**Comment by Cgettys** [comment]
Also, a thought that I just had is that it may be an issue for Nutritionix.com as well, as if you are using the Nutritionix API internally, it gives % daily values.  So if you change the daily values, all of the old % daily values become incorrect.

**Comment by majin22** [comment]
tagging @mattsilv and @Yurko-Fedoriv to keep them in the loop of this conversation

**Comment by PaigeEinstein** [comment]
Hi @Cgettys and thanks for reaching out! We are aware of the changes to the vitamins/minerals that will be taking place on the new nutrition facts label. To summarize them:

**Vitamin D:** new daily value is 20mcg (old value was 400IU). Companies must list mcg and %DV on the label

**Calcium:** new daily value is 1300mg (old value was 1000mg). Companies must list mg and % DV on the label

**Iron:** daily value is unchanged at 18mg. Companies must list mg and %DV on the label.

**Potassium:** new daily value is 4,700mg (old value was 3500mg). Companies must list mg and %DV on the label

**Vitamin C** and **Vitamin A** will not be required on the new label.

Originally, large companies were supposed to comply by next July and smaller companies by July 2019, however this has been delayed. They have not yet announced what the new compliance date is - it could be years. 

Our nutrition team and engineers are working to ensure our data is in compliance with the new nutrition facts label when it does go into effect.

Let me know if you have any other questions or concerns and I'm happy to help!

**Comment by majin22** [comment]
@PaigeEinstein @mattsilv @Yurko-Fedoriv @Cgettys @JCapriotti here are the things that I did for the latest version based on my understanding of your conversation. please let me know if I missed something

1. changed the default unit for `vitamin D` from `IU` to `mcg`

2. added a new `daily potassium value` for the 2018 label with a default value of `4700`. this way, it don't affect the optional potassium value for the legacy label https://github.com/nutritionix/nutrition-label/commit/cff80ba92a399a12d3ba5aa07f651cb751b5c1ff?w=1#diff-9ce72bf5eb39b381c67a00ed3b790dd5

3. changed the `daily calcium value` to `1300`. this value don't affect the legacy version, only the 2018 version

4. changed the `daily vitamin d` value to `20`. this value don't affect the legacy version, only the 2018 version

thanks

**Comment by majin22** [comment]
PS. I created 2 demo pages that uses random values for the attributes that you can use to verify that the computations are correct
* legacy demo http://dev2.nutritionix.com/html/label-jquery-plugin/demo/legacy-version/demo-random-values.html
* 2018 demo http://dev2.nutritionix.com/html/label-jquery-plugin/demo/2018-version/demo-random-values.html

**Comment by Cgettys** [comment]
Thank you so much for the quick turnaround! I did notice one very minor issue - Vitamin D should have a unit of micrograms (mcg), instead of milligrams (mg). I have made a pull request (#76) fixing this issue. Again, thank you for your great work and quick response, and it's been a pleasure working with you.

**Comment by majin22** [comment]
@Cgettys thanks for pointing this out. this is now fixed
